README.md

```diff
@@ -224,6 +224,42 @@ env:
         fieldPath: metadata.name
  ``` 
 
+### Node Aware
```

Comment:
The content of the `Node Aware` section is almost the same as `Zone Aware`. Could you maybe merge both of them and create a section like `Zone / Node Aware`?

Reply:
I know they are almost identical but separate section with own requirements and config is easily digestible. User wants to use Node Aware feature will not be disturbed by Zone Aware requirements or config.

Reply:
Yeah, you're right. One thing that we could improve is to put them both under one section like "High Availability", "Zone/Node Failure Resilience", or "Zone/Node Aware". To have something like:
```
### High Availability
#### Zone Aware
#### Node Aware
```

Anyway, I leave it up to you.